.circleci/config.yml

```diff
@@ -366,7 +366,7 @@ jobs:
               npm run deps:generate
 
               echo "Publish to NPM"
-              npm run tooling -- exec -- bash -c 'npm publish --access public || true'
+              npm run tooling -- exec --cmd bash -- -c 'npm publish --access public || true'
```

Comment:
Tested with `npm run tooling -- exec --cmd bash -- -c 'ls -l'  ` and it behaved properly!